tests/qemu-iotests/iotests.py

```diff
@@ -172,17 +172,17 @@ def add_drive(self, path, opts='', interface='virtio', format=imgfmt):
         if opts:
             options.append(opts)
 
-        self._args.append('-drive')
-        self._args.append(','.join(options))
+        self.args.append('-drive')
+        self.args.append(','.join(options))
         self._num_drives += 1
         return self
 
     def add_blockdev(self, opts):
-        self._args.append('-blockdev')
+        self.args.append('-blockdev')
         if isinstance(opts, str):
-            self._args.append(opts)
+            self.args.append(opts)
         else:
-            self._args.append(','.join(opts))
+            self.args.append(','.join(opts))
```

Comment:
I'm still afraid we might have missed some occurrence, but even my searches failed to find any...